python/nano/setup.py

```diff
@@ -74,43 +74,64 @@ def download_libs(url: str):
                               platform_system=='Darwin'",
                               "keras==2.8.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
                               "tensorflow-estimator==2.8.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]
-    
+
     tensorflow_29_requires = ["intel-tensorflow==2.9.1; (platform_machine=='x86_64' or platform_machine == 'AMD64') and \
                               platform_system!='Darwin'",
                               "tensorflow==2.9.0; platform_machine=='x86_64' and \
                               platform_system=='Darwin'",
                               "keras==2.9.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
                               "tensorflow-estimator==2.9.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]
-    
+
     tensorflow_210_requires = ["intel-tensorflow==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64') and \
                                platform_system!='Darwin'",
                                "tensorflow==2.10.0; platform_machine=='x86_64' and \
                                platform_system=='Darwin'",
                                "keras==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
                                "tensorflow-estimator==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]
-    
+
+    # options for stock tensorflow
+    stock_tensorflow_27_requires = ["tensorflow==2.7.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "keras==2.7.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "tensorflow-estimator==2.7.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]
+
+    stock_tensorflow_28_requires = ["tensorflow==2.8.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "keras==2.8.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "tensorflow-estimator==2.8.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]
+
+    stock_tensorflow_29_requires = ["tensorflow==2.9.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "keras==2.9.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "tensorflow-estimator==2.9.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]                         
+
+    stock_tensorflow_210_requires = ["tensorflow==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "keras==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')",
+                                    "tensorflow-estimator==2.10.0; (platform_machine=='x86_64' or platform_machine == 'AMD64')"]  
+
```

Comment:
for stock tensorflow, should we install the latest patch of each minor version? like 2.7.4, 2.8.4, 2.9.3, 2.10.1.

Reply:
Have updated each minor version to the latest one.

BTW, I noticed that the installation of `tensorflow` and `intel-tensorflow` will install the corresponding version of `keras` and `tensorflow-estimator` automatically. Is there any reason that we manually specify them? Could we remove them?

Reply:
Have removed the manually-added requirements for `keras` and `tensorflow-estimator` as discussion.